deployments/stat20/config/prod.yaml

```diff
@@ -0,0 +1,19 @@
+nfsPVC:
+  nfs:
+    shareName: export/homedirs-other-2020-07-29/stat20/prod
```

Comment:
you need to ssh into the NFS server and create this one.

Reply:
Done, thanks. I figured I'd do this right before signing in the first time.